server/command.go

```diff
@@ -219,28 +219,28 @@ func withParamIssueKey(cmd *model.AutocompleteData) {
 func createConnectCommand() *model.AutocompleteData {
 	connect := model.NewAutocompleteData(
 		"connect", "", "Connect your Mattermost account to your Jira account")
-	connect.AddDynamicListArgument("Jira URL", routeAutocompleteConnect, false)
+	connect.AddDynamicListArgument("Jira URL", routeAutocomplete+routeAutocompleteConnect, false)
 	return connect
 }
 
 func createAliasCommand() *model.AutocompleteData {
 	alias := model.NewAutocompleteData(
 		"alias", "", "Create an alias to your Jira instance")
-	alias.AddDynamicListArgument("Jira URL", routeAutocompleteInstalledInstanceWithAlias, false)
+	alias.AddDynamicListArgument("Jira URL", routeAutocomplete+routeAutocompleteInstalledInstanceWithAlias, false)
```

Comment:
I think we should have a `makeAutocompleteRoute` function instead of duplicating this concatenation logic